kolibri/plugins/facility/assets/src/views/DataPage/SyncInterface/index.vue

```diff
@@ -130,7 +130,7 @@
   import CoreMenu from 'kolibri.coreVue.components.CoreMenu';
   import CoreMenuOption from 'kolibri.coreVue.components.CoreMenuOption';
   import { TaskStatuses } from 'kolibri.utils.syncTaskUtils';
-  import { PageNames } from './../../../constants';
+  import { PageNames } from '../../../../../../device/assets/src/constants';
```

Comment:
We should revert this - the page names for the facility plugin are distinct from the page names for the device plugin.